diff --git a/ios/LinkPreview/LinkPreviewNativeActionView.swift b/ios/LinkPreview/LinkPreviewNativeActionView.swift
--- a/ios/LinkPreview/LinkPreviewNativeActionView.swift
+++ b/ios/LinkPreview/LinkPreviewNativeActionView.swift
@@ -138,6 +138,6 @@ class LinkPreviewNativeActionView: RouterViewWithLogger, LinkPreviewMenuUpdatable
     baseUiAction.state = isOn == true ? .on : .off
 
-    if let subtitle = subtitle {
+    if #available(iOS 16.0, *), let subtitle = subtitle {
       baseUiAction.subtitle = subtitle
     }
     if let label = discoverabilityLabel {
